### PR TITLE
I3S-Debug - Change Semantic Validator UI

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
image
image

### CODE REVIEW

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js [hunk 1]
```diff
@@ -1,89 +1,105 @@
...
-  flex-flow: column nowrap;
+  background-color: white;
+  z-index: 1000;
 `;
 
-const WarningItem = styled.div`
-  margin-left: 5px;
-  color: red;
+const TableHeader = styled.th`
+  position sticky;
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/components/semantic-validator.js:26]: `:` is missed?

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js [hunk 2]
```diff
@@ -1,89 +1,105 @@
...
+    title: 'Geometry vs Textures validation is not implemented!'
+  }
+];
+
+const NO_ISSUES = 'No Issues';
+
+const WARNING_TYPES = {
+  lod: 'LOD metric',
+  boundingVolume: 'Bounding Volume',
+  geometryVsTextures: 'Geometric vs Textures'
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/components/semantic-validator.js:53]: *Geometry

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js [hunk 3]
```diff
@@ -1,89 +1,105 @@
...
-const BOUNDING_VOLUMES_WARNINGS = 'Bounding Volumes';
-const MAX_SCREEN_TRESHOLDS_WARNINGS = 'Max Screen Tresholds';
-const GEOMETRIC_TEXTURES_WARNINGS = 'Geometric and Textures';
-
 export default class SemanticValidator extends PureComponent {
-  renderWarningItem(warnings) {
-    const warnList = warnings
-      ? warnings.map(warning => <WarningItem key={warning}>{warning}</WarningItem>)
-      : [];
+  prepareColumns(warnings) {
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/components/semantic-validator.js:68]: `renderColumns` is more appropriate for React semantics

**Review by belom88 — APPROVED** [review]
A few minor comments.
I suppose we can merge it as `modules` haven't been affected, only debug application.

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/components/semantic-validator.js]: Added

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/components/semantic-validator.js]: done

File: examples/experimental/i3s-17-and-debug/components/semantic-validator.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/components/semantic-validator.js]: renamed

**Pull request merged**